samples/corda/corda-simple-application/build.gradle

```diff
@@ -48,6 +48,35 @@ allprojects {
         mavenCentral()
         maven { url 'https://software.r3.com/artifactory/corda' }
         maven { url 'https://jitpack.io' }
+
+	if (file("$projectDir/github.properties").exists()) {
+            Properties constants = new Properties()
+            file("$projectDir/github.properties").withInputStream { constants.load(it) }
+            maven {
+                url constants.url
+                credentials {
+                    username constants.username
+                    password constants.password
+                }
+            }
+	} else if (file("$projectDir/../github.properties").exists()) {
```

Comment:
Why is this needed?

Reply:
This is to know if the test network is launched using `locally built Weaver components` vs. `imported Weaver components`.